features/admin_locations_create.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Create a new location
+  As a project manager
+  I can create a new location
+  So that I can keep the list of locations up to date
```

Comment:
Not sure you'll agree but my feeling was all of it was superfluous.

The Feature and Scenario names alone give enough context.